pkg/rules/rulespb/custom.go

```diff
@@ -186,6 +186,27 @@ func (r *RuleGroups) MarshalJSON() ([]byte, error) {
 	return json.Marshal((*plain)(r))
 }
 
+// Compare compares rule group x and y and returns:
+//
+//   < 0 if x < y   if rule group r1 is not equal and lexically before rule group r2
+//     0 if x == y  if rule group r1 is logically equal to r2 (r1 and r2 are the "same" rule groups)
+//   > 0 if x > y   if rule group r1 is not equal and lexically after rule group r2
+//
+// For sorting this makes sure that more "critical" alert states come first.
```

Comment:
is this relevant?

Reply:
no it is a copy-paste error, good catch!